build.gradle

```diff
@@ -202,5 +241,43 @@ android {
               systemProperty 'robolectric.logging', 'stdout'
         }
     }
+}
 
+/**
+ * Download and unpack commcare app associated with 'cc_app_id' into assets
+ * folder
+ */
+task downloadCCApp(type: Exec)
```

Comment:
Oh sorry I was thinking about having changed how this worked for parto, but that's just on that branch